terminal/terminal.md

```diff
@@ -0,0 +1,417 @@
+danielvilla@Daniels-MacBook-Air:~/docker$ 
+```
+
+By the way, the **~** before the **$** sign indicates the home directory, all users have their own home directory, and is VERY IMPORTANT that you know that **/** indicates the root directory, off all the filesystem
+
+but now that we are on the docker directory how do we move back to the home directory, there are two ways:
+
+`danielvilla@Daniels-MacBook-Air:~/docker$ cd ..`
+
+and
+
+`danielvilla@Daniels-MacBook-Air:~/docker$ cd`
+
+**cd** without any arguments changes to your home directory no matter where you are
+
+You can also take advantage of the **~** symbol and use it to change to a directory inside your home 
+
+```
+danielvilla@Daniels-MacBook-Air:~/projects/selenium-projects/likeNeitor$ cd ~/docker
+danielvilla@Daniels-MacBook-Air:~/docker$ 
+```
+or use the absoluth path (in case the directory is not in your home)
+
+```
+danielvilla@Daniels-MacBook-Air:~$ cd /var/log/ppp/
+danielvilla@Daniels-MacBook-Air:/var/log/ppp$ 
+```
+
+But now let's say we are working on two very far directories
+`~/projects/react_app/config/` and `/etc/ssh/`
+
+#### More on ls command
+
+We already learn about **ls** but as we saw **ls** only showed us the list of file with no metadata
+
+```
+danielvilla@Daniels-MacBook-Air:~/docker/linux_tweet_app$ ls
+Dockerfile          README.md           index-new.html      index-original.html index.html          linux.png
+```
+
+but as a curious user that I am, I want to know the date the file was created, the size, the owner. To display it lets use the **-l** flag. Yes, we can use flags on commands to make them do more
+
+```
+danielvilla@Daniels-MacBook-Air:~/docker/linux_tweet_app$ ls -l
+total 88
+-rw-r--r--  1 danielvilla  staff    147 Feb 18 20:48 Dockerfile
+-rw-r--r--  1 danielvilla  staff    284 Feb 18 20:48 README.md
+-rw-r--r--  1 danielvilla  staff   1673 Feb 18 20:48 index-new.html
+-rw-r--r--  1 danielvilla  staff   1540 Feb 18 20:48 index-original.html
+-rw-r--r--  1 danielvilla  staff   1595 Feb 18 20:48 index.html
+-rw-r--r--  1 danielvilla  staff  22365 Feb 18 20:48 linux.png
+```
+Awesome, but that's kinda confusing, what are those numbers next to the staff word? Most commands that deal with storage properties share a commond flag **-h** that means "human readable"
+
+```
+danielvilla@Daniels-MacBook-Air:~/docker/linux_tweet_app$ ls -lh
+total 88
+-rw-r--r--  1 danielvilla  staff   147B Feb 18 20:48 Dockerfile
+-rw-r--r--  1 danielvilla  staff   284B Feb 18 20:48 README.md
+-rw-r--r--  1 danielvilla  staff   1.6K Feb 18 20:48 index-new.html
+-rw-r--r--  1 danielvilla  staff   1.5K Feb 18 20:48 index-original.html
+-rw-r--r--  1 danielvilla  staff   1.6K Feb 18 20:48 index.html
+-rw-r--r--  1 danielvilla  staff    22K Feb 18 20:48 linux.png
+```
+Now we can see a more understandable output regarding the file size, and as you might guess Feb 18 is the date when the file was created. Don't worry about the other stuff for now
```

Comment:
Now we can see a more understandable output regarding the file size, and as you might guess Feb 18 is the date when the file was created. Don't worry about the other stuff for now`.`